configure.ac

```diff
@@ -568,13 +568,17 @@ AX_CHECK_COMPILE_FLAG([-march=armv8-a+crc+crypto],[[ARM_CRC_CXXFLAGS="-march=arm
 
 TEMP_CXXFLAGS="$CXXFLAGS"
 CXXFLAGS="$CXXFLAGS $ARM_CRC_CXXFLAGS"
-AC_MSG_CHECKING(for ARM CRC32 intrinsics)
+AC_MSG_CHECKING(for ARM64 CRC32 intrinsics)
```

Comment:
nit: AArch64 is a more accurate name, since it's not really related to the 32-bit ARM architecture AIUI.

Reply:
Sure, fine with changing the name, though what crc32c library uses internally is ARM64 so I mimiced that.